packages/shared/src/contexts/BootProvider.tsx

```diff
@@ -120,6 +121,11 @@ export const BootDataProvider = ({
 
   useEffect(() => {
     if (bootRemoteData) {
+      if (initialLoad === null) {
+        setInitialLoad(true);
+      } else {
+        setInitialLoad(false);
+      }
```

Comment:
Just a simplification.
```suggestion
      setInitialLoad(initialLoad === null);
```

Reply:
Yes, I think that should work as that is the condition itself.